utils/docker/Dockerfile.centos.7

```diff
@@ -87,17 +87,27 @@ RUN pip3 install avocado-framework; \
 RUN pip3 install codespell
 RUN pip3 install pygithub
 
-# Add DAOS user
+# Add DAOS users
+ENV USER daos_server
+ENV PASSWD daos_server
+RUN useradd -u $UID -ms /bin/bash $USER
```

Comment:
UID should be reserved for the daos user and not be reused for daos_server/daos_agent that should have their own (random?) uid.

Reply:
fixed...